Spanner/src/Operation.php

```diff
@@ -187,6 +188,11 @@ public function commitWithResponse(Session $session, array $mutations, array $op
      */
     public function rollback(Session $session, $transactionId, array $options = [])
     {
+        // TODO: Check with team on this and decide
+        if (is_array($transactionId) and isset($transactionId['begin'])) {
+            return;
+            // throw new \RuntimeException('Streaming calls must invoke rows() function');
```

Comment:
remove comment?
```suggestion
```

Reply:
Here having no transactionId means, the rollback has called before even creating the transaction. This can happen in a case where a user initiated a streaming calls but did not fetch the results. Thats why I added the throw exception comment. Then I thought team can have a discussion on it and then remove this comment.

Reply:
When I see commented out logic like this in a PR (without a comment explaining why), I assume it was left in the code unintentionally. If it's intentional, then it would be better to leave a comment explaining why the line is commented out. For example:
```php
// @TODO: discuss with team whether we should throw an exception in some cases
// throw new \RuntimeException('Streaming calls must invoke rows() function');
```
Then we know to revisit it, and it's clear that your intention is to remove or uncomment the line before the PR is merged.